README.md

```diff
@@ -1,6 +1,33 @@
-# New Acropolis UK API  [![Build Status](https://travis-ci.org/NewAcropolis/api.svg?branch=master)](https://travis-ci.org/NewAcropolis/api) 
+# New Acropolis UK API  [![Build Status](https://travis-ci.org/NewAcropolis/api.svg?branch=master)](https://travis-ci.org/NewAcropolis/api)
 [![Coverage Status](https://coveralls.io/repos/github/NewAcropolis/api/badge.svg?branch=master)](https://coveralls.io/github/NewAcropolis/api?branch=master)
 
+## Create virtualenv (optional)
+
+A Virtual Environment is an isolated working copy of Python which
+allows you to work on a specific project without worry of affecting other projects
+
+Follow this guide to set up your virtualenv for this project;
+https://virtualenvwrapper.readthedocs.io/en/latest/
+
+## Install requirements
+
+run `pip install -r requirements.txt`
```

Comment:
Could add this in the `Makefile` and have a reference to the make command eg. `make install`

Reply:
Added a 'make dependencies' which is already in the frontend to install the requirements.txt